libnavui-maneuver/src/main/res/drawable/mapbox_ic_arrive_right.xml

```diff
@@ -1,9 +1,3 @@
-<vector xmlns:android="http://schemas.android.com/apk/res/android"
-    android:height="20dp"
-    android:viewportHeight="20"
-    android:viewportWidth="20"
-    android:width="20dp">
-  <path
-      android:fillColor="?attr/maneuverTurnIconColor"
-      android:pathData="M15,10a2,2,0,1,1,2,2a2,2,0,0,1-2-2Zm-8,1.0549300000000006h1v-.009090000000000487a.35757.35757,0,0,1,.37738.54242l-1.3437500000000009,2.9518400000000007a.3866.3866,0,0,0,.62354.40942l6.34283-4.903680000000001l-6.34283-4.90369a.38659.38659,0,0,0-.62354.40942l1.3437500000000009,2.9518500000000003a.35757.35757,0,0,1-.3773800000000005.5424199999999999v.009090000000000487h-.9918199999999997l.00011999999999989797-.020080000000000098l-.011.00183a6.05607,6.05607,0,0,0-5.98345,5.9633199999999995h-.013849999999999696v1.48767a.51232.51232,0,0,0,.51233.5123299999999986h.9753400000000001a.51232.51232,0,0,0,.51233-.5123299999999986v-1.3777500000000007h.008300000000000196a4.05782,4.05782,0,0,1,3.98877-4.07324" />
```

Comment:
Android Lint says that `.009` in the pathData can cause a crash on some devices. Lint suggests replace `.009` by `0.009`. I tried to fix the errors manually but the found out that we have too many vector drawables.

I used the avocado tool which I found in the [article](https://android-blogs.medium.com/optimize-svgs-remove-lint-warning-of-the-long-vector-path-fe0b34f80782) to automatically fix all paths. Unfortunately avocado also reformats the files.

Reply:
I will try to find a way to configure avocado or use different tool to fix paths.

Reply:
fixed paths using regex